pkg/graph/topological_test.go

```diff
@@ -0,0 +1,59 @@
+package graph
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/dominikbraun/graph"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStableTopologicalOrder(t *testing.T) {
+	assert := assert.New(t)
+
+	g := NewDirected[dummyVertex]()
+	g.AddVerticesAndEdge("a", "c")
+	g.AddVerticesAndEdge("b", "c")
+
+	archetype, err := g.VertexIdsInTopologicalOrder()
+	if !assert.NoError(err) {
+		return
+	}
+	archetypeStr := strings.Join(archetype, "")
+	assert.True(archetypeStr == "abc" || archetypeStr == "bac")
```

Comment:
Shouldn't this be in a single specific ordering to make sure it doesn't flip across compilations?

Reply:
I was trying to convey in the test that it doesn't really matter _which_ it is, as long as it's then the same for the next 10k. I could be even more explicit, to make sure that it's stable not just across invocations but also across processes.

Reply:
https://github.com/klothoplatform/klotho/pull/530